src/openfl/display/BitmapData.hx

```diff
@@ -1766,6 +1770,7 @@ class BitmapData implements IBitmapDrawable {
 			renderSession.clearRenderDirty = true;
 			renderSession.context = cast buffer.__srcContext;
 			renderSession.allowSmoothing = smoothing;
+			renderSession.scale = __scale;
```

Comment:
we should probably add the same to the public `draw` method...

Reply:
True, each RenderSession must store the correct pixelRatio right?